lib/linguist/languages.yml

```diff
@@ -5473,6 +5473,18 @@ Vim script:
   - vimrc
   ace_mode: text
   language_id: 388
+Vim snippet:
+  type: markup
+  aliases:
+  - SnipMate
+  - UltiSnip
+  - NeoSnippet
+  extensions:
+  - ".snip"
+  - ".snippets"
```

Comment:
You can also add [`.snippet`](https://github.com/genghuiluo/vimrc/tree/65badb75815e4b21b941e89879d68ad4dbedb17c/sources_non_forked/snipmate-snippets/ruby-rspec) as an extension, too (no plural).

I got halfway through researching each extension's usage earlier this year before getting interrupted, but I'd already confirmed their distribution is adequate. 👍